zuul.d/projects.yaml

```diff
@@ -391,3 +391,10 @@
     3pci-check:
       jobs:
         - noop
+
```

Comment:
This needs to be added in a 2nd PR, we first need to merge the zuul/tenants.yaml change, for zuul to load its default config.

Reply:
this needs to be removed